pkg/destroy/gcp/instance.go

```diff
@@ -130,7 +130,12 @@ func (o *ClusterUninstaller) stopInstance(ctx context.Context, item cloudResourc
 	o.Logger.Debugf("Stopping compute instance %s in zone %s", item.name, item.zone)
 	ctx, cancel := context.WithTimeout(ctx, defaultTimeout)
 	defer cancel()
-	op, err := o.computeSvc.Instances.Stop(o.ProjectID, item.zone, item.name).RequestId(o.requestID("stopinstance", item.zone, item.name)).Context(ctx).Do()
+	op, err := o.computeSvc.Instances.
+		Stop(o.ProjectID, item.zone, item.name).
+		RequestId(o.requestID("stopinstance", item.zone, item.name)).
+		DiscardLocalSsd(true).
```

Comment:
According to https://cloud.google.com/compute/docs/disks/local-ssd these disks are supposed to offer temporary storage. Since we're stopping the VMs and destroying the cluster, I can't see what negative consequences there could be?

Reply:
Also from that doc:
```
If you want to preserve the data of the Local SSD disks attached to the VM, you must [stop](https://cloud.google.com/compute/docs/instances/stop-start-instance#stop-vm) or [suspend](https://cloud.google.com/compute/docs/instances/suspend-resume-instance#suspend_an_instance) the VM by using the gcloud CLI and including the --discard-local-ssd=false flag. This begins a managed migration of Local SSD data to persistent storage, and you're charged for the additional storage utilization until you [restart the VM](https://cloud.google.com/compute/docs/instances/stop-start-instance#restart-vm). You might have to remount the Local SSD disk into the file system after restarting the VM.
```
https://cloud.google.com/compute/docs/disks/local-ssd#stop_instance

So there is a charge implication by setting that value to `false`.

Reply:
Besides, deleting the VM destroys the ssd disk anyway:

> To remove or delete Local SSD disks, you delete the VM they are attached to. Before you [delete a VM](https://cloud.google.com/compute/docs/instances/deleting-instance) with Local SSD disks, make sure that you migrate any critical data from the Local SSD disk to a Persistent Disk, Hyperdisk, or to another VM.

https://cloud.google.com/compute/docs/disks/local-ssd#remove_local_ssd_disks